elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -197,7 +202,8 @@ async def get_bugs_sweep(runtime: Runtime, find_bugs_obj, brew_event, bug_tracke
 
 
 async def find_and_attach_bugs(runtime: Runtime, advisory_id, default_advisory_type, major_version,
-                               find_bugs_obj, output, brew_event, noop, count_advisory_attach_flags, bug_tracker):
+                               find_bugs_obj, output, brew_event, noop, count_advisory_attach_flags, bug_tracker,
+                               cve_only: bool):
```

Comment:
is the parameter `cve_only` used in function `find_and_attach_bugs`?

Reply:
No, it is a leftover that has been moved to the constructor of `FindBugsSweep`.
Will remove it in the next push.